ls1intum/Artemis
# `Development`: Cleanup documentation

### Checklist
#### General


- [x] This is a small issue that I tested on the readthedocs platform. I did review it myself there.

### Motivation and Context


I found one broken link during another task and wanted to fix it.
Took this as an opportunity to also trim the line length of the other .rst files for better readability!

### Description

One commit fixes the broken links.
One commit fixes the line lengths.

### Steps for Testing


- check if links of commit one are working
- check that formatting of the modified .rst files didn't change

#### Code Review
#### Manual Tests
